test/integration/scheduler_perf/config/performance-config.yaml

```diff
@@ -35,6 +35,12 @@
       initNodes: 5000
```

Comment:
Should we just remove `performance` tag from the existing medium ones (`5000Nodes`)?
If it's too noisy like you said, I don't think it's worth keeping in perf-dash.

Reply:
Good idea, removed